packages/app/pages/course/[cid]/today.tsx

```diff
@@ -30,6 +30,24 @@ const Title = styled.div`
   font-size: 30px;
   color: #212934;
 `;
+const RoleColorSpan = styled.span`
+  color: #3684c6;
+  font-weight: bold;
+`;
+
+function roleToString(role: Role) {
```

Comment:
i feel like i've seen a similar logic elsewhere. maybe we should centralize this mapping somewhere lm aO

Reply:
_withdraw_ pr flashbacks :flooshed:

Reply:
ahhhhh